tests/test_repair.py

```diff
@@ -57,7 +57,6 @@ def test__find_intact_sub_trees():
     obj._fill_repair_type_map()
     assert_raises(Exception, obj._find_intact_sub_trees)
```

Comment:
I am curious why this line passes if you dropped this exception `Exception('No intact basal dendrites !')`

Reply:
Lol, it was failing for a different reason. A list was passed were a numpy array was expected. Thanks for spotting this !